public/jira/rollup/percent-complete/percent-complete.test.ts

```diff
@@ -0,0 +1,813 @@
+            },
+            {
+              completedWorkingDays: 0,
+              remainingWorkingDays: 20,
+              totalWorkingDays: 20,
+              userSpecifiedValues: true,
+            },
+          ],
+        },
+        {
+          rollupData: [
+            {
+              completedWorkingDays: 0,
+              remainingWorkingDays: 30,
+              totalWorkingDays: 30,
+              userSpecifiedValues: false,
+            },
+            {
+              completedWorkingDays: 0,
+              remainingWorkingDays: 30,
+              totalWorkingDays: 30,
+              userSpecifiedValues: true,
+            },
+          ],
+        },
+        {
+          rollupData: [
+            {
+              completedWorkingDays: 0,
+              remainingWorkingDays: 1,
+              totalWorkingDays: 1,
+              userSpecifiedValues: true,
+            },
+          ],
+        },
+      ];
+
+      const actual = rollupPercentComplete(issuesAndReleases);
+
+      expect(actual).toHaveLength(expected.length);
+      for (const i of Array(actual.length).keys()) {
+        expect(actual[i].rollupData).toStrictEqual(expected[i].rollupData);
+      }
+    });
+    it("should use average estimates and completion if no estimate provided for stories", () => {
+      const issuesAndReleases = (
+        [
+          [
+            {
+              key: "i-1",
+              parentKey: null,
+              derivedTiming: { totalDaysOfWork: 1, completedDaysOfWork: 0 },
+            },
+          ],
+          [
+            {
+              key: "e-2",
+              parentKey: "i-1",
+              derivedTiming: { totalDaysOfWork: null, completedDaysOfWork: 5 },
+            },
+            {
+              key: "e-3",
+              parentKey: "i-1",
+              derivedTiming: { totalDaysOfWork: 2000, completedDaysOfWork: 100 },
+            },
+          ],
+          [
+            {
+              key: "s-4",
+              parentKey: "e-2",
+              derivedTiming: { totalDaysOfWork: null, completedDaysOfWork: 2 },
+            },
+            {
+              key: "s-5",
+              parentKey: "e-2",
+              derivedTiming: { totalDaysOfWork: null, completedDaysOfWork: 4 },
+            },
+            {
+              key: "s-6",
+              parentKey: "e-3",
+              derivedTiming: { totalDaysOfWork: 10, completedDaysOfWork: 8 },
+            },
+            {
+              key: "s-7",
+              parentKey: "e-3",
+              derivedTiming: { totalDaysOfWork: 20, completedDaysOfWork: 16 },
+            },
+          ],
+        ] as IssueOrRelease[][]
+      ).reverse();
+
+      const expected = [
+        {
+          rollupData: [
+            {
+              completedWorkingDays: 0,
+              remainingWorkingDays: 15,
+              totalWorkingDays: 15,
+              userSpecifiedValues: false,
+            },
+            {
+              completedWorkingDays: 0,
```

Comment:
This seems to be ignoring completed working days .... Does this mean the code is breaking?  I'd expect this to be 4 and the one above 2.